src/content_cao.cpp

```diff
@@ -360,6 +360,10 @@ v3f GenericCAO::getPosition()
 	return pos_translator.vect_show;
 }
 
+const int GenericCAO::isImmortal(){
```

Comment:
no need for `const` here
should return `bool`
code style

Reply:
I'm not sure if immortal will always be bool.

Reply:
The group can have a different value sure, but the question this function answers is "Is this ActiveObject immortal?" to which you want a "Yes" or "No", not some number.